qiskit/visualization/matplotlib.py

```diff
@@ -990,16 +996,22 @@ def _draw_ops(self, verbose=False):
                     self._ctrl_qubit(q_xy[1], fc=ec, ec=ec, tc=tc)
                     self._line(qreg_b, qreg_t, lc=lc, zorder=PORDER_LINE + 1)
 
-                # cu1, rzz, and controlled rzz gates (sidetext gates)
-                elif (op.name == 'cu1' or op.name == 'rzz' or base_name == 'rzz'):
+                # cu1, cp, rzz, and controlled rzz gates (sidetext gates)
+                elif (op.name == 'cu1' or op.name == 'cp'
+                      or op.name == 'rzz' or base_name == 'rzz'):
```

Comment:
```suggestion
                elif op.name in ['cu1', 'cp', 'rzz'] or base_name == 'rzz':
```

Reply:
Done in https://github.com/Qiskit/qiskit-terra/pull/5804/commits/54073ddf01439a371ed126e03296a4c61e9bffa3